docs/README.md

```diff
@@ -52,3 +52,22 @@ rails test test/path/to/the_test.rb:123
 ## Deployment
 
 While we await provisioning of appropriate services on the VA network, we are hosting the platform-console on Heroku. Each commit to master is automatically deployed and each pull request will generate a Review App.
+## KEYCLOAK INTEGRATION
```

Comment:
Could we extract this to a file in docs/?

Reply:
Yes, we definitely can.

Reply:
Please do. :D